docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -139,13 +139,17 @@ Preparing For a Release
 
       git push --set-upstream origin $(git rev-parse --abbrev-ref HEAD)
 
-#. Ensure all the required tests pass on BuildBot:
+#. Ensure all the required tests pass on Jenkins:
 
-   Pushing the branch in the previous step should have started a build on BuildBot.
-   If not, you can force a build by logging in to BuildBot, entering the release branch name in to the box at the top right and clicking the ``Force`` button.
+   To run the tests on `Jenkins`_, first run `setup_ClusterHQ-flocker-release <http://ci-live.clusterhq.com:8080/job/setup_ClusterHQ-flocker-release/>`_ using the release branch as the parameter to the job.
+   This will generate two sets of test jobs for the release branch which can be accessed from the `releases view <http://ci-live.clusterhq.com:8080/job/ClusterHQ-flocker/view/releases/>`_.
+   For the following steps, use the results of the jobs within "Release release/flocker-<VERSION>" as these do not perform any pre-build merging with master.
+
+   To run the tests, force a build of the "__main_multijob" job.
```

Comment:
Put `_main_multijob` inside double-backquotes to avoid adding a non-word to the spelling list.